app/manager-frontend/src/pages/Agent/list.jsx

```diff
@@ -0,0 +1,5 @@
+export const AgentsList = () => {
+  // load Agents data from backend
+  // display Agents List
+  return <></>;
+};
```

Comment:
This doesn't seem to be doing anything.

Reply:
Yep. This will be included in the next PR as a part of Agent List task (AB#84786), which includes populating agents list from the backend data.